test/integer.test.js

```diff
@@ -83,3 +85,10 @@ test('render an object with an additionalProperty of type integer as JSON', (t)
   t.equal(output, '{"num":1615}')
   t.ok(validate(JSON.parse(output)), 'valid schema')
 })
+
+if (semver.gt(process.versions.node, '10.3.0')) {
+  require('./bigint')(t.test, build)
+} else {
+  t.pass('Skip because Node version < 10.4')
```

Comment:
Shouldn’t there be at least one test here to show the behavior on old versions?

Reply:
I fear that I’m missing something. Can you propose a test?
As I said above, this is a good old typeof, which will work in any Node version.
In version ma older than 10.4 you can’t use the BigInt syntax.